storage/storage.go

```diff
@@ -1315,6 +1315,27 @@ type ObjectAttrs struct {
 	// later value but not to an earlier one. For more information see
 	// https://cloud.google.com/storage/docs/metadata#custom-time .
 	CustomTime time.Time
+
+	// As of 2015-06-03, the official GCS documentation for this
```

Comment:
This is too much detail for public docs, and we are not faking a component count of 1 in this library. You can just give a short summary of what component count is instead.

I would say something like: `ComponentCount is the number of objects contained within a composite object. For non-composite objects, the value will be zero. This field is read-only.`